### PR TITLE
DL 137 

### PR DESCRIPTION
**Pull request by ericmoritz**
[DL 137](https://jira.gannett.com/browse/DL-137)

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
 source 'https://gdp-chef-supermarket.gannettdigital.com'
 
+cookbook 'users'
```
Comment (fhanson) [inline Berksfile]: you don't need these if they are defined in the metadata

File: metadata.rb [hunk 2]
```diff
@@ -4,6 +4,10 @@
 license          'All rights reserved'
 description      'Installs/Configures mapr_installation'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.2.10'
+version          '0.3.0'
 
+depends 'users'
```
Comment (fhanson) [inline metadata.rb]: It might be wise to add specific versions here

**Comment by dwashko** [comment]
This cookbook needs a .kitchen.ec2.yml file. You can pull an example from here: https://confluence.gannett.com/display/GDPINT/3+-+Test+Kitchen

Pull down the .kitchen.ec2.yml file and edit accordingly, paying particular attention to copying what you did in .kitchen.yml suites section.

**Comment by ericmoritz** [comment]
Is this good to go?

**Pull request merged**